pom.xml

```diff
@@ -529,6 +529,19 @@
     <!--</site>-->
   <!--</distributionManagement>-->
```

Comment:
Shall we delete this existing commented distributionManagement element if not needed after this?